### ISSUE TITLE
Write initial usage docs for `@carbon/styles`

### ISSUE THREAD

**Issue by joshblack** [carbon-design-system/carbon#9169]
This issue is for writing out the initial usage docs for the `@carbon/styles` package. These usage docs should include:

- [ ] How to bring in the package and use the styles
- [ ] How to bring in individual component styles
- [ ] How to change the theme
- [ ] How to use design tokens (overlap with Design Tokens)
- [ ] How to bring in IBM Plex to your project

These docs should be present in the `README.md` and `docs/sass.md` files in the `packages/styles` folder. The `README.md` should contain high-level, common information and the `docs/sass.md` file will be comprehensive/contain all information.

In other words, `README.md` should contain high-level usage/examples. `docs/sass.md` should contain detailed API information
